rewrite_test.php

```diff
@@ -14,8 +14,7 @@
 <head>
     <title>Rewrite Test &bull; NamelessMC</title>
 
-    <!-- Global CSS -->
-    <link rel="stylesheet" href="core/assets/css/bootstrap.min.css">
+    <link rel="stylesheet" href="core/assets/vendor/bootstrap/dist/js/bootstrap.min.js">
```

Comment:
Is the change from `.css` to `.js` intentional here

Reply:
Haha, no. Good catch